### PR TITLE
Add possibleFutureConnections to Schedule

### PR DESCRIPTION
**Pull request by lisajulia**
Description is here: https://github.com/OPM/opm-simulators/pull/5488

### CODE REVIEW

File: opm/input/eclipse/Schedule/Action/PyAction.cpp [hunk 1]
```diff
@@ -40,7 +40,7 @@ namespace Action {
 bool PyAction::valid_keyword(const std::string& keyword) {
     static std::unordered_set<std::string> pyaction_allowed_list = {
         "BOX",
-        "COMPSEGS",
+        "COMPDAT", "COMPLUMP", "COMPSEGS",
```
Comment (blattms) [inline opm/input/eclipse/Schedule/Action/PyAction.cpp:43]: Shoud this be part of this PR or another one?

**Review by blattms — COMMENTED** [review]
Looks good except for the PYACTION changes that are included here.

**Comment by lisajulia** [comment]
jenkins build this opm-tests=1199 opm-grid=742 opm-simulators=5488 please

**Comment by lisajulia** [comment]
jenkins build this opm-tests=1199 opm-grid=742 opm-simulators=5488 please

**Comment by lisajulia** [comment]
jenkins build this opm-tests=1200 opm-grid=742 opm-simulators=5488 please

**Comment by blattms** [comment]
Thanks for the heads up. I just will double check that this also resolves OPM/opm-simulators#5494

**Comment by blattms** [comment]
Wait a minute: Jenkins checked that yesterday with "jenkins build this opm-tests=1199 opm-grid=742 opm-simulators=5488 please", right?

**Comment by lisajulia** [comment]
> jenkins build this opm-tests=1200 opm-grid=742 opm-simulators=5488 please

and today with this

**Pull request merged**